sources/web/datalab/polymer/templates/NewNotebookTemplate.ipynb

```diff
@@ -0,0 +1,43 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
```

Comment:
I thought we were going to have two markdown cells, one with the header title and one with a one line description?

Reply:
Also, are we going to make the title a parameter?